### DISCUSSION TITLE
"open token.json: Access is denied."

### DISCUSSION THREAD

**Discussion by fourcandy** [bedrock-tool/bedrocktool#99] [Q&A]
So I'm trying to use bedrocktool but it keeps saying this after signing in:
"open token.json: Access is denied."

There is no token.json where the app is situated. I deleted it the first time it said this.

**Comment by olebeck** [comment]
try putting it in a folder that is on the desktop, specifically not downloads

**Reply by fourcandy** [reply]
OK I'll try it
